src/utils/i18n/translations/en-US.json

```diff
@@ -0,0 +1,18 @@
+{
+  "unauthorized.message": "You have no permission to view or modify it.",
+  "common.optionalFieldLabel": "(Optional)",
+  "common.add": "Add",
+  "common.refreshPage": "Refresh Page",
+  "common.more": "More",
+  "loading.loading": "Loading",
```

Comment:
Just minor thing, does the loading component have the other translation other than `Loading`? If it doesn't have it better to use as `common.loading`, wdyt?

Reply:
i actually just copied it from AP. but that sounds good too